app/res/layout/setup_keepinstall.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+    <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                    xmlns:SquareButtonWithText="http://schemas.android.com/apk/res/org.commcare.dalvik"
+                    android:id="@+id/screen_first_start_main"
+                    android:layout_width="fill_parent"
+                    android:layout_height="fill_parent"
+                    android:orientation="vertical"
+                    android:background="@color/cc_core_bg"
+        >
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
+            <!--&lt;!&ndash; TODO: extract style definition &ndash;&gt;-->
+            <ImageView
+                android:id="@+id/screen_first_start_banner"
```

Comment:
It really seems like these two fragments are very similar, and that it's possible that there should be one host fragment (with the banner and stuff) and one internal fragment with the buttons, which would make this workflow a bit easier to extend. 

Might seem less relevant once the dimensions and stuff are pulled out a bit, though?